heartbeat/status/export_test.go

```diff
@@ -0,0 +1,22 @@
+package status
+
+// NewMetricsUpdaterWithoutGoRoutineStart -
+func NewMetricsUpdaterWithoutGoRoutineStart(args ArgsMetricsUpdater) (*metricsUpdater, error) {
+	err := checkArgs(args)
+	if err != nil {
+		return nil, err
+	}
+
+	updater := &metricsUpdater{
+		peerAuthenticationCacher:            args.PeerAuthenticationCacher,
+		heartbeatMonitor:                    args.HeartbeatMonitor,
+		heartbeatSenderInfoProvider:         args.HeartbeatSenderInfoProvider,
+		appStatusHandler:                    args.AppStatusHandler,
+		timeBetweenConnectionsMetricsUpdate: args.TimeBetweenConnectionsMetricsUpdate,
+		heartbeatV1DisableEpoch:             args.HeartbeatV1DisableEpoch,
+	}
+
+	args.EpochNotifier.RegisterNotifyHandler(updater)
```

Comment:
```suggestion
	args.EpochNotifier.RegisterNotifyHandler(updater)
	args.PeerAuthenticationCacher.RegisterHandler(updater.onAddedPeerAuthenticationMessage, "metricsUpdater")
```

otherwise `TestMetricsUpdater_MetricLiveValidatorNodesUpdatesDirectly` passes only because it never checks the conditions

Reply:
right, thanks.

Reply:
fixed tests + export_test.go